pennylane/tape/jacobian_tape.py

```diff
@@ -91,6 +91,7 @@ class JacobianTape(QuantumTape):
     def __init__(self, name=None, do_queue=True):
         super().__init__(name=name, do_queue=do_queue)
         self.jacobian_options = {}
+        self.hessian_options = {}
```

Comment:
```suggestion
```

`hessian_options` isn't actually needed, as `jacobian_options` will be a catch-all for differentiation options. 

Maybe we should update the name?